docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -803,25 +803,35 @@ in the following sections:
 
 === Spring Cloud LoadBalancer Caching
 
-Apart from the basic `ServiceInstanceListSupplier` implementation that retrieves instances
-via `DiscoveryClient` each time it has to choose an instance, we provide a https://github.com/ben-manes/caffeine[Caffeine-backed]
-implementation.
-To make use of it, you need to have `com.github.ben-manes.caffeine:caffeine` in the classpath.
-The default setup includes `expireAfterWrite` set to 30 seconds and records set to soft references.
+Apart from the basic `ServiceInstanceListSupplier` implementation that retrieves instances via `DiscoveryClient` each time it has to choose an instance, we provide two caching implementations.
 
-You can set your own `TTL` value (the time after write after which entries should be expired), expressed as `Duration`, by passing a `String` compliant with the https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration[Spring Boot `String` to `Duration` converter syntax].
-as the value of `spring.cloud.loadbalancer.cache.ttl` property.
+==== https://github.com/ben-manes/caffeine[Caffeine]-backed LoadBalancer Cache Implementation
+If you have `com.github.ben-manes.caffeine:caffeine` in the classpath, Caffeine-based implementation will be used.
 
-You can also override the default Caffeine Cache setup for the LoadBalancer by passing your own https://static.javadoc.io/com.github.ben-manes.caffeine/caffeine/2.2.2/com/github/benmanes/caffeine/cache/CaffeineSpec.html[Caffeine Specification]
+If you are using Caffeine, you can also override the default Caffeine Cache setup for the LoadBalancer by passing your own https://static.javadoc.io/com.github.ben-manes.caffeine/caffeine/2.2.2/com/github/benmanes/caffeine/cache/CaffeineSpec.html[Caffeine Specification]
 in the `spring.cloud.loadbalancer.cache.caffeine.spec` property.
 
-WARN: Passing your own Caffeine specification will override any other LoadBalancerCache settings, including `TTL`.
+WARN: Passing your own Caffeine specification will override any other LoadBalancerCache settings, including <<loadbalancer-cache-configuration, General LoadBalancer Cache Configuration>> fields, such as `ttl` and `capacity`.
+
+==== Default LoadBalancer Cache Implementation
+
+If you do not have Caffeine in the classpath, but have https://github.com/stoyanr[`com.stoyanr:evictor`] (which comes automatically with `spring-cloud-starter-loadbalancer`), the default, Evictor-based, implementation will be used.
```

Comment:
> If you do not have Caffeine in the classpath, but have https://github.com/stoyanr[`com.stoyanr:evictor`] (which comes automatically with `spring-cloud-starter-loadbalancer`)

I would leave all this stuff about Evictor out.  Just tell them how to 

1.  Configure the caching parameters
2. Use Caffeine instead of our own cache implementation